packages/visualizations/src/components/Legend/types.ts

```diff
@@ -25,17 +25,22 @@ export interface LegendConfiguration {
 
 export type LegendVariant = 'fluid' | 'fixed';
 
+type ItemVariant = 'box' | 'circle' | 'line';
+
 export type CategoryItem = {
     color?: Color;
+    variant?: ItemVariant;
     borderColor?: Color;
     borderDashed?: boolean;
     label: LegendLabelsConfiguration | string | undefined;
-    onClick: (index: number) => void;
+    onClick?: (index: number) => void;
     onHover?(index: number, isVisible: boolean): void;
     onLeave?(): void;
 };
 
 export type CategoryLegend = {
     type: 'category';
     items: CategoryItem[];
+    title?: string;
+    alignement?: 'start' | 'center';
```

Comment:
If `LegendConfiguration` allow 'start', 'center' and 'end' should if property does the same ?